topics/atom/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: atom-package, atom-packages, atom-theme
+created_by: GitHub
+display_name: Atom
+github_url: https://github.com/atom
+logo: atom.png
+released: February 26, 2014
+short_description: Atom is a modern open source text editor built with web technologies.
+topic: atom
+url: https://atom.io/
+wikipedia_url: https://en.wikipedia.org/wiki/Atom_(text_editor)
+---
+Atom is a modern open source text editor developed by GitHub. It is designed to be approachable yet customisable with a wide variety of options and packages.
```

Comment:
Should probably go with the American "customizable" spelling given GitHub is a US-based company.

Reply:
@damieng Done in af925138fd1da822f985c292ff57e125bac35bb2